.gitignore

```diff
@@ -109,10 +109,17 @@ venv.bak/
 # PyCharm settings
 .idea
 
+# Terraform
+.terraform
+*.tfstate*
+
+
 # Certs
 **/*.cert
 **/*.key
 **/*.pem
+**/certs/party_key.txt
```

Comment:
Optional: perhaps we should just ignore the folder as a whole

Reply:
I think in the future this will probably be removed as the party key will be passed in as part of the config manager, I'll leave this for the time being since regardless of where the certs are we will always want to ignore them.